src/pages/extensions/index.tsx

```diff
@@ -132,6 +133,11 @@ const ExtensionsPage = () => {
         downloadLink="https://github.com/Testaustime/testaustime-micro"
         sourceCodeLink="https://github.com/Testaustime/testaustime-micro"
         text={t("extensions.micro")}
+      /><ExtensionBlock
+        logo={<Sublime width={40} height={40} />}
+        downloadLink="https://github.com/Testaustime/testaustime-sublime"
```

Comment:
Should this be a link to the Github releases page? I don't know how installing Sublime extensions works though, so maybe I'm wrong

```suggestion
        downloadLink="https://github.com/Testaustime/testaustime-sublime/releases/latest"
```